ui_components/src/home/Home.js

```diff
@@ -375,32 +378,70 @@ function LoginSection({
+                                <svg role="button" tabIndex="0" name="login" className="login_button" width="162" height="32" onClick={() => handleLogin()} style={{ zIndex: 2 }}>
+                                    <radialGradient id="login_fill_default" cx="50%" cy="50%" r="50%" fx="50%" fy="50%">
+                                        <stop offset="0%" style={{ stopColor: '#464f87', stopOpacity: 1 }} />
+                                        <stop offset="100%" style={{ stopColor: '#343d77', stopOpacity: 1 }} />
+                                    </radialGradient>
+                                    <radialGradient id="login_fill_click" cx="50%" cy="50%" r="50%" fx="50%" fy="50%">
+                                        <stop offset="0%" style={{ stopColor: '#343d77', stopOpacity: 1 }} />
+                                        <stop offset="100%" style={{ stopColor: '#464f87', stopOpacity: 1 }} />
+                                    </radialGradient>
+                                    <rect className="login_button_background" width="100%" height="100%" fill="url(#login_fill_default)"/>
+                                    <text className="login_button_shadow_text" x="81" y="18" textAnchor="middle" dominantBaseline="middle">login</text>
+                                    <text className="login_button_text" x="81" y="16" textAnchor="middle" dominantBaseline="middle">login</text>
+                                </svg>
+                                <svg role="button" tabIndex="0" name="register" className="register_button" width="162" height="32" onClick={() => handleRegister()} style={{ zIndex: 4 }}>
+                                    <radialGradient id="register_fill_default" cx="50%" cy="50%" r="50%" fx="50%" fy="50%">
+                                        <stop offset="0%" style={{ stopColor: '#84314e', stopOpacity: 1 }} />
+                                        <stop offset="100%" style={{ stopColor: '#68293f', stopOpacity: 1 }} />
+                                    </radialGradient>
+                                    <radialGradient id="register_fill_click" cx="50%" cy="50%" r="50%" fx="50%" fy="50%">
+                                        <stop offset="0%" style={{ stopColor: '#68293f', stopOpacity: 1 }} />
+                                        <stop offset="100%" style={{ stopColor: '#84314e', stopOpacity: 1 }} />
+                                    </radialGradient>
+                                    <rect className="register_button_background" width="100%" height="100%" />
+                                    <text className="register_button_shadow_text" x="81" y="18" textAnchor="middle" dominantBaseline="middle">create a character</text>
+                                    <text className="register_button_text" x="81" y="16" textAnchor="middle" dominantBaseline="middle">create a character</text>
+                                </svg>
+                            </>
+                        }
```

Comment:
With these blocks of code, it's easier to work with if they are their own React components (can keep it in the same file, but move the render code itself) out of this component.